Solutions/Corvus.Configuration.TestEnvironment/Corvus/Configuration/ConfigurationExtensions.cs

```diff
@@ -39,6 +39,7 @@ public static class ConfigurationExtensions
         /// }
         /// .
         /// </remarks>
+        [Obsolete("Use AddConfigurationForTest() instead.")]
         public static void AddTestConfiguration(this IConfigurationBuilder builder, string filePath, IDictionary<string, string> fallbackSettings = null)
```

Comment:
Do we need anything in the comments explaining how this came to be obsoleted?

Reply:
Have added a reference to this PR.